docs/hsi.xml

```diff
@@ -161,7 +168,8 @@
       <para>
         This security level corresponds to the most basic of security protections
         considered essential by security professionals.
-        Any failures at this level would have critical security impact.
+        Any failures at this level would have critical security impact and could
+        likely be used to compromise the system firmware without opening the case.
```

Comment:
Rather than focusing on the case, how about "physical access"

Reply:
That's much better, thanks.